### PR TITLE
Create Montotown.json

### LINKED ISSUE #7272: I'm a robot
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by Montotown**
No comments!!!
@githubteacher 
closes #7272

### CODE REVIEW

**Comment by mathol** [comment]
This is AMAZEBALLEZ !!!
:+1: 

WHY AM I SCREAMING OMG

**Comment by sudipju** [comment]
great job !

**Comment by bas** [comment]
>WHY AM I SCREAMING OMG

@Montotown no idea why you were screaming. May be you like loud music? 🎸  🤘

**Comment by sudipju** [comment]
amazing though :+1:

File: _pins/Montotown.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Montotown
```
Comment (sudipju) [inline _pins/Montotown.json]: excellent :100:

Comment (Montotown) [inline _pins/Montotown.json]: Wow!

File: _pins/Montotown.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+latitude: 59.3247
+longitude: 18.056
```
Comment (Montotown) [inline _pins/Montotown.json]: Hubba

File: _pins/Montotown.json (hunk 1 above)
Comment (mathol) [inline _pins/Montotown.json]: This is great, especially in single user mode. However,  when will we support stereo?

**Pull request merged**